libs/coin-evm/src/api/explorer/ledger.ts

```diff
@@ -126,8 +127,29 @@ export const getLastOperations: ExplorerApi["getLastOperations"] = async (
   };
 };
 
+const getTransactionByHash = async (
+  currency: CryptoCurrency,
+  transactionHash: string,
+): Promise<{ confirmations?: number }> => {
+  const { explorer } = currency.ethereumLikeInfo || /* istanbul ignore next */ {};
```

Comment:
```suggestion
  const { explorer } = currency.ethereumLikeInfo!;
```
I could be wrong but both have the same result

Reply:
Nope, the non-null assertion will result in an error thrown if `ethereumLikeInfo` is undefined, while the `|| {}` allow to throw (or not) in an expected manner later on the flow.